quickstarts/dotnet/blazor-server/config.yml

```diff
@@ -0,0 +1,48 @@
+slug: blazor-server
+
+description: |
+  ## Comprehensive monitoring quickstart for Blazor Server
+  Microsoft developed Blazor Server to build web UIs with C#, HTML, and CSS. As a feature of ASP.NET, our Blazor Server integration with the .NET agent lets you monitor your apps with our quickstart dashboards right out of the box.
+
+  ## Why monitor Blazor Server?
+  Blazor Server offers a lot of useful features in addition to allowing C# developers to utilize their preferred language for web development. Blazor Server allows you to run your code through Blazor Server on the server or the client, and you can decide which users or devices experience each method.
+  
+  There are a few compelling arguments that support the employment of Blazor Server:
+  - All of the major browsers support Blazor Server.
+  - Using C# for interactive web apps.
+  - Utilize current C# libraries.
+  - Near-native performance.
+  - Debugging and tooling.
```

Comment:
Delete all of this and rewrite to focus on why you should monitor a Blazor Server with New Relic. This is all about why you'd use Blazor Server, not why you'd use New Relic to monitor it.